packages/rx_bloc_cli/mason_templates/bricks/lib_mfa/__brick__/bin/server/controllers/mfa_controller.dart

```diff
@@ -1,19 +1,20 @@
 {{> licence.dart }}
 
 import 'package:shelf/shelf.dart';
-import 'package:{{project_name}}/lib_auth_matrix/methods/otp/models/auth_matrix_otp_payload.dart';
-import 'package:{{project_name}}/lib_auth_matrix/methods/pin_biometric/models/auth_matrix_pin_code_payload.dart';
-import 'package:{{project_name}}/lib_auth_matrix/models/auth_matrix_action.dart';
-import 'package:{{project_name}}/lib_auth_matrix/models/auth_matrix_method.dart';
-import 'package:{{project_name}}/lib_auth_matrix/models/auth_matrix_method_request.dart';
-import 'package:{{project_name}}/lib_auth_matrix/models/auth_matrix_response.dart';
-import 'package:{{project_name}}/lib_auth_matrix/models/payload/response/auth_matrix_last_login_payload_response.dart';
+import 'package:{{project_name}}/lib_mfa/methods/otp/models/mfa_otp_payload.dart';
+import 'package:{{project_name}}/lib_mfa/methods/pin_biometric/models/mfa_pin_code_payload.dart';
+import 'package:{{project_name}}/lib_mfa/models/payload/response/mfa_last_login_payload_response.dart';
+import 'package:{{project_name}}/lib_mfa/models/mfa_action.dart';
+import 'package:{{project_name}}/lib_mfa/models/mfa_method.dart';
+import 'package:{{project_name}}/lib_mfa/models/mfa_method_request.dart';
+import 'package:{{project_name}}/lib_mfa/models/mfa_response.dart';
+
 
 import '../utils/api_controller.dart';
 import '../utils/server_exceptions.dart';
 
-class AuthMatrixController extends ApiController {
-  AuthMatrixController();
+class MFAController extends ApiController {
```

Comment:
According to the [Effective Dart guidelines](https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words), abbreviations of more than 2 letters in class names should be capitalised like regular words.

So in this case it should be `MfaController`, other instances of the abbreviation should be updated too

Reply:
Good catch.